palindromic-number-generator.js

```diff
@@ -1,5 +1,24 @@
 function palindromicNumberGenerator(num) {
-  // do work here
+  function reverseNum(input) {
+    return parseInt(
+      input
+        .toString()
+        .split('')
+        .reverse()
+        .join(''),
+    );
+  }
+  let steps = 0;
+  while (num !== reverseNum(num)) {
```

Comment:
It's best to assign the value that comes out of reverseNum to a variable and just use the variable twice rather than call the function twice.